### PR TITLE
pull request 

### PR DESCRIPTION
**Pull request by Deadmole007**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,5 +33,6 @@ Microsoft is participating in Hacktoberfest this year! Details [here](https://op
 | Aditya Vyas | II | IT |
 | Rishabh Anand | IV | ECE | 
 | Shiv Soni | IV | CSE |
+| Harshit Choudhary | II | IT |
```
Comment (amritsrivastava) [inline README.md]: @Deadmole007 please add your name in the middle don't add it at the end or beginning of the file.

Comment (maddhruv) [reply README.md]: how will that effect?
conflicts!?

Comment (amritsrivastava) [reply README.md]: I recommended this as it is mentioned in the NOTE at end of the README. 
Don't know the actual reason though.

Comment (Man-Jain) [reply README.md]: Even I dont know how this works but it does. :sweat_smile:

**Pull request merged**